Sources/ArgumentDescription.swift

```diff
@@ -29,19 +29,28 @@ extension ArgumentConvertible {
 
 open class VariadicArgument<T : ArgumentConvertible> : ArgumentDescriptor {
   public typealias ValueType = [T]
+  public typealias Validator = (T) throws -> T
```

Comment:
I think the validator should take `ValueType` (Array of Value) instead of each individual Value. You might wish to validate the set together, for example the amount of them, or the values for uniqueness.

What do you think about changing the validator to be an array of values?